### ISSUE TITLE
[Screen Reader-Project-Makecode Documentation] Name Property and Control type is not defined for the 'Search Icon' showing near Search edit box. 

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2804]
**User Experience:**
Users with Visual disabilities might not able to understand the controls if Name Property and control type is not defined. Like in this scenario Name Property and control type is not defined for the Search icon.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1- Navigate to https://makecode.microbit.org/projects
2- Navigate to the Search icon showing near Search editable box on project documentation page.
3-Verify that Name Property and Control type is defined for 'Search Icon' or not.

**Actual Result**
Name Property and Control type is not defined for the 'Search Icon' showing near Search edit box as Screen Reader reads it as Text on scan mode and blank in Items mode.

**Expected Result**
Name Property and Control type should be defined for the 'Search Icon' showing near Search edit box.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.1.2%20%E2%80%93%20Name,%20Role,%20Value.docx?d=w248054a65e6847719e1e242fb5025730&csf=1

**Suggested fix:**
1- Using aria-label to give an invisible name to element, where name/title property is not used 
e.g.  
'Close' button can have only ‘X’, to make it more meaningful aria-label attribute is used with ‘Close’ text to help AT’s without showing the content on UI 
 
`<div id="box">` 
   This is a pop-up box. 
     `<button aria-label="Close" onclick="document.getElementById('box').style.display='none';" class="closebutton">` X `</button>`                                          
`</div> `
 
2-   Using ARIA role to expose the control type of user interface component 
e.g. 
`<div role="toolbar" tabindex="0" id="customToolbar" >  `
`<img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home"> `
  
**Refer Attachment:**
![mas4 1 2_makecodde_project_name and control type of search icon](https://user-images.githubusercontent.com/30434417/29665655-afef7d4c-88f2-11e7-83e5-6f9c0cd86aae.png)

**Comment by veler** [comment]
This issue is duplicated [https://github.com/Microsoft/pxt/issues/2625](https://github.com/Microsoft/pxt/issues/2625). It has been fixed but is not deployed yet.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL:https://makecode.microbit.org/projects and issue is fixed now. Hence closing the bug.
